sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -44,23 +45,38 @@ internal class RadarActivityLifecycleCallbacks(
     }
 
     override fun onActivityResumed(activity: Activity) {
-        if (count == 0) {
+        Log.w(TAG, "ON RESUME ${count} ${isFirstCreate}")
```

Comment:
I don't think we need to merge this logging statement?

Reply:
testing, I'll remove this